event-handlers/Slate/CBL/Tasks/Attachments/_/afterRecordSave/sync-google-attachment-permissions.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Slate\CBL\Tasks\Attachments;
+
+use Slate\People\Student;
+use Slate\CBL\Tasks\StudentTask;
+
+$Attachment = $_EVENT['Record'];
+
+// student attached a drive file
+if ($Attachment->isNew && $Attachment->Context && $Attachment->Context->isA(StudentTask::class) && $Attachment->Creator && $Attachment->Creator->isA(Student::class)) {
+    // note: DISABLED UTNIL GOOGLE DRIVE BATCH API IS UPDATED
```

Comment:
Not sure if this change belongs upstream (disabling google drive sync while API is currently deprecated)

Reply:
@nbey yeah upstream